.github/workflows/run_tests.yml

```diff
@@ -7,7 +7,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ["macos-latest"]
-        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
not too important but we should be on 3.12 by now, latest is 3.12.7

Reply:
Thanks Ell, I should have done this on a separate PR really. I've brought a jira card in to look into upgrading the project to 3.12 and will pick it up once this is merged

Reply:
Sweet, thank you :)

Reply:
I think the biggest breaking changes are 3.10 and 3.13, so if we're migrating across those it's probably a bigger job - though it tends to affect specific libraries that I don't think we have in the cookiecutter